lib/index.js

```diff
@@ -1,4 +1,5 @@
 /* @flow */
+/* eslint-disable */
```

Comment:
Never ever completely disable ESLint, or entire rules for that matter.

You generally should be using this if you _need_ to disable a specific rule for a specific bit of code:
```js
// eslint-disable-next-line rule-name
var foo = 1; // code that violates rule-name
```

For details, see [the ESLint documentation](https://eslint.org/docs/user-guide/configuring#disabling-rules-with-inline-comments).

If an entire rule should be disabled for the entire project (very unlikely), it should be done in the ESLint configuration in `package.json`.